packages/model-viewer/package.json

```diff
@@ -76,35 +73,34 @@
     "@google/model-viewer-shared-assets": "^0.0.1",
     "@jsantell/event-target": "^1.1.2",
     "@magicleap/prismatic": "^2.0.2",
-    "@polymer/paper-button": "^3.0.1",
-    "@polymer/paper-radio-group": "^3.0.1",
-    "@polymer/paper-slider": "^3.0.1",
     "@types/chai": "^4.2.7",
     "@types/mocha": "^5.2.5",
     "@types/pngjs": "^3.4.0",
     "@ungap/event-target": "^0.1.0",
     "@webcomponents/webcomponentsjs": "~2.1.3",
     "chai": "^4.1.2",
-    "chokidar-cli": "^1.2.1",
-    "concurrently": "^4.0.1",
-    "cross-os": "^1.3.0",
     "focus-visible": "^5.0.1",
     "fullscreen-polyfill": "^1.0.2",
+    "http-server": "^0.12.1",
```

Comment:
Better than local-web-server?

Reply:
Yes, and consistent with the other packages